.eslintrc.js

```diff
@@ -2,13 +2,14 @@
 
 module.exports = {
   root: true,
-  parser: 'babel-eslint',
+  parser: '@babel/eslint-parser',
   parserOptions: {
     ecmaVersion: 2018,
```

Comment:
We can maybe update the ECMA version to a more recent one.
```suggestion
    ecmaVersion: 2021,
```